lab_1/main.py

```diff
@@ -4,56 +4,168 @@
 """
 
 
-def tokenize(text: str) -> list:
-    """
+#data = open('data.txt', encoding='utf-8').read()
+data = 'The weather is sunny, the man is happy. man man happy'
+def tokenize(text):
+    if type(text) != str:
+        return []
+    else:
```

Comment:
you don't need "else", if "return []" works, this function will finish its work right away